code/config/lib/Loot/Config/Record.hs

```diff
@@ -53,15 +55,17 @@ module Loot.Config.Record
        , selection
        ) where
 
-import Data.Default (Default (..))
-import Data.Validation (Validation (Failure, Success), toEither)
-import Data.Vinyl (Label, Rec ((:&), RNil))
-import Data.Vinyl.Lens (RecElem, rlens, rreplace, type (<:))
-import Data.Vinyl.TypeLevel (RIndex)
-import GHC.TypeLits (AppendSymbol, ErrorMessage ((:<>:), ShowType, Text),
-    KnownSymbol, Symbol, symbolVal, TypeError)
-
-import qualified Text.Show (Show (show))
+import           Data.Default         (Default (..))
+import           Data.Validation      (Validation (Failure, Success), toEither)
+import           Data.Vinyl           (Label, Rec (RNil, (:&)))
+import           Data.Vinyl.Lens      (RecElem, rlens, rreplace, type (<:))
+import           Data.Vinyl.TypeLevel (RIndex)
+import           GHC.TypeLits         (AppendSymbol,
+                                       ErrorMessage (ShowType, Text, (:<>:)),
+                                       KnownSymbol, Symbol, TypeError,
+                                       symbolVal)
+
+import qualified Text.Show            (Show (show))
```

Comment:
Please do not reformat imports in existing code.

Reply:
sorry, it's all about missing config in this repo, I didn't noticed it